example/package.json

```diff
@@ -3,21 +3,32 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "node node_modules/react-native/local-cli/cli.js start",
-    "test": "jest"
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint ."
   },
   "dependencies": {
     "native-base": "^2.8.0",
-    "react": "16.4.1",
-    "react-native": "0.56.0",
-    "react-native-fbads": "file:..",
-    "react-native-router-flux": "^4.0.1"
+    "react": "16.13.1",
+    "react-native": "0.63.4",
+    "react-native-fbads": "^6.3.1",
```

Comment:
Could you please let it point to the local version again? Otherwise we can't test yet-unpublished (local) changes in the example app